moveit_ros/planning_interface/test/subframes_test.cpp

```diff
@@ -86,7 +86,7 @@ void spawnCollisionObjects(moveit::planning_interface::PlanningSceneInterface& p
 {
   const std::string log_name = "spawn_collision_objects";
   double z_offset_box = .25;  // The z-axis points away from the gripper
-  double z_offset_cylinder = .1;
+  double z_offset_cylinder = .12;
```

Comment:
Why did you change that value back?
For me, this change was required to have the cylinder centered between the jaws.

Reply:
When I tested locally this change caused it to fail the motion plan because it was in collision.  I posted the log output in your PR.

Reply:
Do you use an outdated panda_moveit_config or franka_description locally?
As https://travis-ci.com/github/ubi-agni/moveit/jobs/305268241 shows, the tests pass with 0.1.

Reply:
```ros.moveit_core.collision_detection.fcl: Found a contact between 'cylinder' (type 'Robot attached') and 'panda_rightfinger' (type 'Robot link'), which constitutes a collision. Contact information is not stored.
ros.moveit_core.collision_detection.fcl: Collision checking is considered complete (collision was found and 0 contacts are stored)
ros.moveit_ros_planning: Start state appears to be in collision with respect to group panda_arm
ros.moveit_ros_planning: Unable to find a valid state nearby the start state (using jiggle fraction of 0.050000 and 100 sampling attempts). Passing the original planning request to the planner.
ros.moveit_planners_ompl.planning_context_manager: Using 'JointModel' parameterization for solving problem
ros.moveit_planners_ompl.planning_context_manager: Creating new planning context
ros.moveit_planners_ompl.model_based_planning_context: panda_arm: Setting planning volume (affects SE2 & SE3 joints only) to x = [-5.000000, 5.000000], y = [-5.000000, 5.000000], z = [-5.000000, 5.000000]
ros.moveit_core.kinematic_constraints: The desired rotation matrix for link 'panda_hand' in frame world is:
 0.000398163           -1  7.40105e-16
          -1 -0.000398163  -6.9279e-12
  6.9279e-12  2.01833e-15           -1
ros.moveit_core.kinematic_constraints: The desired rotation matrix for link 'panda_hand' in frame world is:
 0.000398163           -1  7.40105e-16
          -1 -0.000398163  -6.9279e-12
  6.9279e-12  2.01833e-15           -1
ros.moveit_planners_ompl.constrained_goal_sampler: Constructed a ConstrainedGoalSampler instance at address 0x7fd8700131d0
ros.moveit_planners_ompl.model_based_planning_context: Planner configuration 'panda_arm' will use planner 'geometric::RRTConnect'. Additional configuration parameters will be set when the planner is constructed.
ros.moveit_planners_ompl.planning_context_manager: panda_arm: New planning context is set.
ros.moveit_planners_ompl.model_based_planning_context: panda_arm: Solving the planning problem once...
ros.moveit_planners_ompl.ompl: panda_arm/panda_arm: Skipping invalid start state (invalid state)
ros.moveit_planners_ompl.ompl: panda_arm/panda_arm: Motion planning start tree could not be initialized!
ros.moveit_planners_ompl.ompl: No solution found after 0.000572 seconds
ros.moveit_planners_ompl.ompl: Goal sampling thread never did any work.
ros.moveit_planners_ompl.model_based_planning_context: There were 0 valid motions and 0 invalid motions.
ros.moveit_planners_ompl.model_based_planning_context: Unable to solve the planning problem
ros.moveit_ros_planning_interface.move_group_interface: Fail: ABORTED: No motion plan found. No execution attempted.
ros.moveit_ros_planning_interface: Failed to perform motion.
```

Tested again locally and this is what happens on mine when I change it to 0.1.  Note that in the tutorial this test is based off the value is 0.12.

Reply:
> Do you use an outdated panda_moveit_config or franka_description locally?
> As https://travis-ci.com/github/ubi-agni/moveit/jobs/305268241 shows, the tests pass with 0.1.

If you go to the commit hash linked on that travis job it ran with 0.12.  See: https://github.com/ubi-agni/moveit/blob/a18d278014375764360c78626d53cdc0a0cfa14d/moveit_ros/planning_interface/test/subframes_test.cpp

Reply:
> Do you use an outdated panda_moveit_config or franka_description locally?

That was it!  I didn't realize there was a source install for a newer panda_moveit_config called out in the rosinstall.  Did that happen recently?  Anyway, it is now working for me locally at 0.1 and I updated it to be 0.1.